format-specs/spatial-type-guidance.md

```diff
@@ -0,0 +1,328 @@
+  such an improvement whose definition lives in a non-spatial format is more likely to
+  cause confusion and/or misinterpretation than deferring to an existing standard.
+- **Leverage the capabilities of the format**: Every format has a unique set of
+  features and limitations and there is sometimes a tension between deferring to existing
+  standards and ensuring that users of the format can leverage existing tools that work with
+  the format. For example, [GeoJSON](https://geojson.org) encodes geometries as JavaScript
+  objects instead of a text or binary-based format.
+
+For more details and ways to ensure that the type definition is precise and unambiguous,
+keep reading!
+
+## Prior art
+
+GeoParquet is far from the first attempt to formalize the representation of spatial
+data to an existing type system/storage format. A non-exhaustive list of established
+Spatial Type implementations (in no particular order) we will refer to in this document
+includes:
+
+- [PostGIS](https://postgis.net/) (PostgreSQL)
+- [GeoParquet](https://geoparquet.org) (Parquet)
+- [GeoPandas](https://geopandas.org) (pandas)
+- [Apache Sedona](https://sedona.apache.org) (Apache Spark)
+- [Esri Shapefile](https://doc.arcgis.com/en/arcgis-online/reference/shapefiles.htm)
+  (DBase).
+- [R/sf](https://r-spatial.org/sf) (R)
+- [GeoJSON](https://geojson.org/) (JSON)
+- [GeoPackage](https://www.geopackage.org/) (SQLite)
+- [SpatiaLite](https://www.gaia-gis.it/fossil/libspatialite/index) (SQLite)
+- [Apache Calcite Spatial](https://calcite.apache.org/docs/spatial.html) (Apache Calcite)
+- [BigQuery Geography](https://cloud.google.com/bigquery/docs/reference/standard-sql/geography_functions) (BigQuery)
+- [Snowflake Geospatial](https://docs.snowflake.com/en/sql-reference/data-types-geospatial) (Snowfake)
+- [Redshift Geospatial](https://docs.aws.amazon.com/redshift/latest/dg/geospatial-functions.html) (Redshift)
+- [FlatGeoBuf](http://flatgeobuf.org/) (Flatbuffers)
+- [GeoProtobuf](https://github.com/geo-grpc/api) (gRPC/Protocol buffers)
+- [cuspatial](https://docs.rapids.ai/api/cuspatial/stable/) (RAPIDS/cudf)
+- [GeoZarr](https://github.com/zarr-developers/geozarr-spec) (Zarr)
+- [Climate and Forecasting (CF) Conventions for Geometries](https://cfconventions.org/cf-conventions/cf-conventions.html#geometries)
+
+In addition to established examples of spatial types, several newer libraries are
+in the process of adding spatial type support that we will also refer to in this document:
+
+- [GeoArrow](https://geoarrow.org) (Apache Arrow)
+- [DuckDB Spatial](https://github.com/duckdb/duckdb_spatial) (DuckDB)
+- [Ibis Spatial](https://ibis-project.org/reference/expression-geospatial) (Ibis)
+- [Substrait](https://github.com/substrait-io/substrait/blob/67f93b654e4cc60340a214ba90fe15c5e9de941b/extensions/functions_geometry.yaml)
+
+Some examples of projects that are not considered Geospatial type implementations for
```

Comment:
Above you refer to "Spatial" type and here "Geospatial" type; suggest making those consistent.  Also, the casing is a bit inconsistent (e.g., line 91).